.envrc

```diff
@@ -5,8 +5,5 @@ use nix
 mkdir -p .nix-bundler
 export BUNDLE_PATH=./.nix-bundler
 
-# hook for rbenv locally
-which rbenv &> /dev/null && eval "$(rbenv init - -zsh)"
```

Comment:
I added this previously and decided it's not the best idea so I'm removing it here